src/sql/workbench/services/queryEditor/browser/queryEditorService.ts

```diff
@@ -87,8 +87,10 @@ export class QueryEditorService implements IQueryEditorService {
 		let docUri: URI = URI.from({ scheme: Schemas.untitled, path: filePath });
 
 		// Create a sql document pane with accoutrements
-		const fileInput = this._untitledEditorService.create({ untitledResource: docUri, mode: 'sql' });
+		const fileInput = this._untitledEditorService.create({ associatedResource: docUri, mode: 'sql' });
```

Comment:
>associatedResource [](start = 57, length = 18)

Is there a reason you changed it to untitledResource from associatedResource? Just wanted to know why you changed it in the first place.

Reply:
It was changed to that because it was used to fix the issue of the edit data editor opening up upon reload even when no changes were made. I was told take a similar approach to how it was fixed for notebooks, which used untitledResource to fix the problem, which worked since the dirty state for an untitledResource editor is false. However, I did not realize there was a side effect as you pointed out.

but as you can see, it introduced a side effect even